erpnext/controllers/accounts_controller.py

```diff
@@ -1235,6 +1236,12 @@ def validate_and_delete_children(parent, data):
 
 		d.cancel()
 		d.delete()
+		
+		from erpnext.stock.stock_balance import update_bin_qty, get_reserved_qty, get_ordered_qty
+		update_bin_qty(d.item_code, d.warehouse, {
+			"reserved_qty": get_reserved_qty(d.item_code, d.warehouse),
+			"ordered_qty": get_ordered_qty(d.item_code, d.warehouse)
+		})
```

Comment:
updating both will be time consuming, update it based on the doctype. reserved qty if sales order, otherwise ordered qty